app/scripts/models/user.js

```diff
@@ -89,12 +107,16 @@ define([
     },
 
     getSignedInAccount: function () {
```

Comment:
Should [`getChooserAccount` use `getSignedInAccount` instead of `_getSignedInAccount`](https://github.com/mozilla/fxa-content-server/blob/issue-2921-account-fetch/app/scripts/models/user.js#L148) now?